tests/loaddata/test_office_access.py

```diff
@@ -1,11 +1,13 @@
 from numpy import testing
```

Comment:
since you've renamed office_access to image_access, should this module be renamed as well for consistency?

Reply:
Yes. I will rename it in the next PR. The changes cannot be displayed if I rename it now.